### DISCUSSION TITLE
Make copying or cloning a View possible

### DISCUSSION THREAD

**Discussion by Patrick010** [home-assistant/frontend#17562] [Dashboard]
Would be nice if it were possible to copy or clone a View into a new one, just like can be done with a Card. Maybe this is already possible, but haven't found how to do it.
I have rooms which have identical sensors and automations. Making a new View based on another one would save me quite some work.

**Comment by Mariusthvdb** [comment]
Ill up this

With the new (2024.05) Area dashboard, I was struggling to use the main Home view, as it contains all and grinds the view/dashboard.

Made me want to move it to the far right, and use my own. Basically I wanted to use the same, but without the nested card grids.

Therefor, I opened the edit menu, and hoped to find a Copy button. 

Unfortunately not there.

<img width="601" alt="Scherm­afbeelding 2025-04-03 om 14 12 32" src="https://github.com/user-attachments/assets/a678ed8e-fc40-49dd-9033-ff0610616d3c" />

Had to manually break a day now, and managed in the end. I am aware I could have copied the Yaml, but that is very probe to errors, and lets not speak of editing the `lovelace.dashboard_areas`...

SO, please revive this, and hope the team will consider adding this 'simple' COPY button in the editing menu.
